docs/tech-reference/rsync-distributor.rst

```diff
@@ -0,0 +1,105 @@
+=====================
+RPM rsync Distributor
+=====================
+
+Purpose:
+========
+The RPM rsync distributor publishes RPM content to a remote server. As the name suggests, the
+distributor uses rsync to copy.
+
+Configuration
+=============
+Here's example of rpm_rsync_distributor configuration:
+
+.. code-block:: json
+
+    {
+     "distributor_id": "rpm_rsync_distributor",
+     "distributor_type_id": "rpm_rsync_distributor",
+     "distributor_config": {
+        "remote": {
+            "auth_type": "publickey",
+            "ssh_login": "foo",
+            "key_path": "/home/user/.ssh/id_rsa",
+            "host": "192.168.121.1",
+            "root": "/home/foo/pulp_root_dir"
+        },
+        "predistributor_id": "yum_distributor",
+        "force_full": true
+        }
+    }
+
+``auth_type``
+  Two authentication methods are supported: ``publickey`` and ``password``.
```

Comment:
This implies that rsync is communicating with the target host over SSH. However, as @mhrivnak notes, that's not necessarily true. rsync can copy files between locations on the local filesystem, and it can use other transports. It could be good to mention that SSH is used as the transport.